controllers/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -1642,7 +1699,7 @@ func TestGenerateUnreadyNotificationContent(t *testing.T) {
 		require.Len(t, meta, 4)
 		content, err := GenerateUnreadyNotificationContent(ClusterURLs(toolchainStatus), meta)
 		require.NoError(t, err)
-		require.Len(t, content, 1512)
+		require.Len(t, content, 1522)
```

Comment:
I guess that counting the length of the string is not so important. We could have this instead:
```suggestion
		require.NotEmpty(t, content)
		assert.Contains(t, content, "<h4>Host Operator Deployment not ready</h4>")
		assert.Contains(t, content, "<h4>Member Routes member-sandbox.ccc.openshiftapps.com not ready</h4>")
		assert.Contains(t, content, "<h4>Registration service deployment not ready</h4>")
```

Reply:
Agree , it makes the test a bit more flexible , applied please see: https://github.com/codeready-toolchain/host-operator/pull/836/commits/54d573742cad4df097d7652a5ae06a356182acf0